nucliadb_models/nucliadb_models/link.py

```diff
@@ -63,6 +64,7 @@ class LinkField(BaseModel):
     uri: str
     language: Optional[str] = None
     localstorage: Optional[Dict[str, str]] = {}
+    css_selector: Optional[str] = None
```

Comment:
@Ciniesta one last comment: Can you add pydantic field annotations so that the new parameter is documented on the api?

Reply:
done :)